include/ppx/knob.h

```diff
@@ -54,16 +54,15 @@ class Knob
     const std::string& GetFlagName() const { return mFlagName; }
     const std::string& GetFlagHelp() const { return mFlagDesc; }
     size_t             GetIndent() const { return mIndent; }
+    const std::string& GetHelpParams() const { return mHelpParams; }
```

Comment:
`GetHelpParams` does not sound very descriptive. It would help to have a more descriptive name or at least a comment. Perhaps we could also group the three flag functions together and give them related names. `GetFlagHelp` above is also a bit confusing, and potentially misleading as well (since there was `GetFlagHelpText` below.
Perhaps `GetFlagName`, `GetFlagParameters`/`GetFlagValues` and `GetFlagDescription`?